datasketch/experimental/aio/lsh.py

```diff
@@ -0,0 +1,385 @@
+import sys
+
+if sys.version_info < (3, 6):
+    raise ImportError("Can't use AsyncMinHash module. Python version should be >=3.6")
+
+import asyncio
+import pickle
+from itertools import chain
+from datasketch.experimental.aio.storage import (async_ordered_storage, async_unordered_storage, )
+
+from datasketch.lsh import _optimal_param
+from datasketch.storage import unordered_storage, _random_name
+
+
+class AsyncMinHashLSH(object):
+    """
+    Asynchronous MinHashLSH index.
+
+    :param float threshold: see :class:`datasketch.MinHashLSH`.
+    :param int num_perm: see :class:`datasketch.MinHashLSH`.
+    :param weights: see :class:`datasketch.MinHashLSH`.
+    :type weights: tuple(float, float)
+    :param tuple params: see :class:`datasketch.MinHashLSH`.
+    :param dict storage_config: New types of storage service (aioredis, aiomongo) to use for storing hashtables and keys are implemented. If storage_config is None aioredis storage will be used.
+    :param bool prepickle: for redis type storage use bytes as keys.
+    :param byte base_name: basename. Useful if you want pickle LSH class and spread it into different workers.
+
+    Example:
```

Comment:
The two examples are showing the synchronous usage of the AsyncMinHashLSH, because the use of `await`. Am I right? If so I think we should be showing some asynchronous usages.